src/main/java/com/openlattice/authorization/EdmAuthorizationHelper.java

```diff
@@ -102,10 +104,10 @@ public EdmAuthorizationHelper( EdmManager edm, AuthorizationManager authz ) {
                                 principals ) ) );
     }
 
-    public Set<UUID> getAuthorizedPropertiesOnEntitySet(
+    public Set<UUID> getAuthorizedPropertiesOnNormalEntitySet(
```

Comment:
I thought it would be useful to have a version of this that can take either a normal or a linked entity set and return its set of authorized property types so that whenever this is needed, you don't need to special case linked entity sets vs normal ones (basically the same way `getAuthorizedPropertiesOnNormalEntitySets` has a parallel that's entity-set-type-agnostic)

Reply:
Good idea, I will start to refactor the calls to a general one

Reply:
:+1: